### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by ARhaman** [isaacmacedo/prediction_porosity_campos#1]
Could u please share part of the dataset?

**Comment by Isaac-Nilberto** [comment]
Yes, I will go the link of data set in my drive. I will go to link in a few
minutes. Do you work of geophysics? Where are you from?

Em sáb., 20 de mai. de 2023 09:47, Abdulrahman Ahmed Ahmed Al-Fakih <
***@***.***> escreveu:

**Comment by ARhaman** [comment]
Hello Issac,

Thank you for your email and kind response.

Yes, I'm doing my PhD programs at kfupm,Saudi Arabia

Nice to know you.
